docs/guides/error-mitigation-and-suppression-techniques.ipynb

```diff
@@ -207,11 +208,142 @@
+    "\n",
+    "The following snippet provides an example where PEA is used to mitigate the result of an Estimator job:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4947f442",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "estimator = Estimator(mode=backend)\n",
+    "estimator.options.resilience.zne_mitigation = True\n",
+    "estimator.options.resilience.zne.amplifier = \"pea\"\n",
+    "\n",
+    "estimator.run(pubs, precision=0.01)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aae27947",
+   "metadata": {},
+   "source": [
+    "When choosing to use PEA, Estimator automatically performs the noise reconstruction to prepare for the actual mitigation experiment. The results of this \"noise learning\" stage are reported in Estimator's result metadata. The results can be reused in subsequent experiments, potentially leading to significantly improved runtimes:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d6083ba5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "estimator = Estimator(mode=backend)\n",
+    "estimator.options.resilience.zne_mitigation = True\n",
+    "estimator.options.resilience.zne.amplifier = \"pea\"\n",
+    "\n",
+    "# The first job performs the noise reconstruction in preparation to the actual estimation task.\n",
+    "job1 = estimator.run(pubs, precision=0.01)\n",
+    "\n",
+    "# The noise data is available in the results.\n",
+    "noise_data = job1.result().metadata[\"resilience\"][\"layer_noise_model\"]\n",
+    "\n",
+    "# It can be reused in subsequent estimation jobs to skip the noise-learning stage.\n",
+    "estimator.options.resilience.layer_noise_model = noise_data\n",
+    "job2 = estimator.run(pubs, precision=0.01)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "59ab042a",
+   "metadata": {},
+   "source": [
+    "The noise learning stage can also be performed independently of the estimation by using the `NoiseLearner` class:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9ab50448",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "learner = NoiseLearner(backend)\n",
+    "learner_job = learner.run(pubs)  # `run` accepts pubs or a list of circuits\n",
+    "noise_data = job.result()\n",
+    "\n",
+    "estimator = Estimator(mode=backend)\n",
+    "estimator.options.resilience.zne_mitigation = True\n",
+    "estimator.options.resilience.zne.amplifier = \"pea\"\n",
+    "estimator.options.resilience.layer_noise_model = noise_data\n",
+    "estimator_job = estimator.run(pubs, precision=0.01)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "68268837",
+   "metadata": {},
+   "source": [
+    "<Admonition type=\"note\"> \n",
+    "`EstimatorV2.run` invokes noise learning each time it is called.  Therefore, to ensure that you are using a consistent noise model, it is important to only call `run()` once. \n",
```

Comment:
This feels like out of place here. I would remove it, together with the examples of where the noise learning is run twice